specification/cognitiveservices/data-plane/Speech/TextToSpeech/preview/2024-02-01-preview/texttospeech.json

```diff
@@ -0,0 +1,2630 @@
+        },
+        "capabilities": {
+          "description": "Capabilities of base model",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ModelCapability"
+          }
+        }
+      }
+    },
+    "Consent": {
+      "description": "Consent object",
+      "required": [
+        "voiceTalentName",
+        "companyName",
+        "locale",
+        "projectId"
+      ],
+      "type": "object",
+      "properties": {
+        "id": {
+          "$ref": "#/definitions/ResourceId"
+        },
+        "displayName": {
+          "description": "Name of consent.",
+          "minLength": 1,
+          "type": "string"
+        },
+        "description": {
+          "description": "Description of consent.",
+          "type": "string"
+        },
+        "voiceTalentName": {
+          "description": "Voice talent name. Must match the voice talent name in the consent audio file.",
+          "minLength": 1,
+          "type": "string"
+        },
+        "companyName": {
+          "description": "Company name. Must match the company name in the consent audio file.",
+          "minLength": 1,
+          "type": "string"
+        },
+        "audioUrl": {
+          "description": "The public accessible URL of the consent audio file. It's recommended to be an Azure blob URL with [SAS](https://learn.microsoft.com/azure/storage/common/storage-sas-overview). This property is only available in request.",
+          "type": "string",
+          "format": "uri"
+        },
+        "locale": {
+          "description": "The locale of this consent. Locale code follows BCP-47. You can find the text to speech locale list here https://learn.microsoft.com/azure/ai-services/speech-service/language-support?tabs=tts.",
+          "type": "string",
+          "example": "en-US"
+        },
+        "projectId": {
+          "$ref": "#/definitions/ResourceId"
+        },
+        "properties": {
+          "$ref": "#/definitions/ConsentProperties"
+        },
+        "status": {
+          "$ref": "#/definitions/Status"
+        },
+        "createdDateTime": {
+          "description": "The timestamp when the object was created. The timestamp is encoded as ISO 8601 date and time format (\"YYYY-MM-DDThh:mm:ssZ\", see https://en.wikipedia.org/wiki/ISO_8601#Combined_date_and_time_representations)",
+          "format": "date-time",
+          "type": "string",
+          "readOnly": true
+        },
+        "lastActionDateTime": {
+          "description": "The timestamp when the current status was entered. The timestamp is encoded as ISO 8601 date and time format (\"YYYY-MM-DDThh:mm:ssZ\", see https://en.wikipedia.org/wiki/ISO_8601#Combined_date_and_time_representations).",
+          "format": "date-time",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ConsentFailureReason": {
+      "description": "Consent creation failure reason",
+      "enum": [
+        "None",
```

Comment:
Is this `None` mean "consent does not fail" or "we don't know why it fails"?

Reply:
The 2nd one. consent failed, but we don't know why.

Reply:
If service decide to keep the `None`, you may consider at least add `description` part in [x-ms-enum](https://azure.github.io/autorest/extensions/#x-ms-enum) to document it.